connect/clients/nats.py

```diff
@@ -42,8 +42,8 @@ async def create_nats_subscribers():
     Create NATS subscribers.  Add additional subscribers as needed.
     """
     await start_sync_event_subscribers()
-    await start_timing_subscriber()
-    await start_retransmit_subscriber()
+    await start_subscriber("TIMING", nats_timing_event_handler)
```

Comment:
Should we add a config for TIMING ?

Reply:
We decided (at the time) to make internal subjects literals and not configurable.  We can revisit if needed.

Reply:
Gotcha. That make sense. We don't really want to override these with an environment variable. Thanks for the reminder!